rmw_fastrtps_cpp/src/ros_message_serialization.cpp

```diff
@@ -20,15 +20,20 @@
 bool
 _serialize_ros_message(
   const void * ros_message,
+  eprosima::fastcdr::FastBuffer & buffer,
```

Comment:
can you call `reserve` on the internal buffer of `eprosima::fastcdr::Cdr` ?

Reply:
Yes. Only when FastBuffer is created with it's default constructor and before serialization has started (i.e. internal buffer has not been allocated yet). 

https://github.com/eProsima/Fast-CDR/blob/fbed79e854769a28d58280ada630bfb98f546a01/include/fastcdr/FastBuffer.h#L266-L271